generators/php/sdk/versions.yml

```
@@ -1,11 +1,26 @@
+- version: 0.1.2
+  changelogEntry:
+    - type: feat
+      summary: >-
+        Represent enums in objects as strings.
+    - type: fix
+      summary: >-
+        Generated wrapped requests now implement `SerializableType`.
+        Represent enums in objects as strings.
```

Let's remove this (I know you meant to based on the entry above).
```suggestion
        Generated wrapped requests now implement `SerializableType`.
```

---

oop thanks!